docs/latest/design_docs.rst

```diff
@@ -10,4 +10,8 @@ Learn about the internal details of Envoy Gateway.
   design/gatewayapi-translator
   design/watching
   design/config-api
-  design/tcp-udp-design
\ No newline at end of file
+  design/tcp-udp-design
+  design/egctl.md
```

Comment:
I don't think we need the `.md` extensions in the links. If we remove them we would make it consistent with the rest of the list.